DataFormats/Detectors/TOF/src/CalibTimeSlewingParamTOF.cxx

```diff
@@ -168,3 +189,14 @@ CalibTimeSlewingParamTOF& CalibTimeSlewingParamTOF::operator+=(const CalibTimeSl
   }
   return *this;
 }
+//______________________________________________
+CalibTimeSlewingParamTOF::CalibTimeSlewingParamTOF(const CalibTimeSlewingParamTOF& source)
+{
+  bind();
+  for (int i = 0; i < NSECTORS; i++) {
+    *(mTimeSlewing[i]) = *(source.mTimeSlewing[i]);
+    (*(mChannelStart[i])) = (*(source.mChannelStart[i]));
+    *(mFractionUnderPeak[i]) = *(source.mFractionUnderPeak[i]);
+    *(mSigmaPeak[i]) = *(source.mSigmaPeak[i]);
```

Comment:
don't you need to copy also mGlobalOffset?

Reply:
You are right. I probably missed it when copy and paste (and workflow didn't complain also in execution).

About #7610
I implemented an autodetection for fill scheme in case it is not provided.
It will be automatically skipped once the fill scheme will be filled with the information of collision context (at the moment I didn't know how to retrieve that info from there).

Reply:
fixed